java/sage/plugin/CorePluginManager.java

```diff
@@ -1946,97 +1967,67 @@ else if ("FanArt".equals(packages[i].type))
           return "FAILED - Invalid Package Type " + packages[i].type;
         }
 
-        java.io.File currFile = (java.io.File) packageMap.get(packages[i]);
-        // We need to check for overwrite at each file if its OK or not; we also need to stage any failed extractions
-        // zip file source, extract and preserve directory structure, permissions and such
-        if (Sage.DBG) System.out.println("Beginning zip file extraction from cache file " + currFile + " to root dest dir of: " + destDir);
-        java.util.zip.ZipFile zippy = null;
-        java.io.InputStream inStream = null;
-        java.io.FileOutputStream outStream = null;
         byte[] buf = new byte[65536];
-        try
+
+        java.io.File currFile = (java.io.File) packageMap.get(packages[i]);
+        if ("JAR".equals(packages[i].type) && currFile.getName().endsWith("jar"))
```

Comment:
Oh, well in that case just leave it alone. I forgot how that worked. :)

On Tue, Sep 6, 2016 at 4:51 PM, Sean Stuckless <notifications@github.com>
wrote:

> In java/sage/plugin/CorePluginManager.java
> <https://github.com/google/sagetv/pull/169#discussion_r77739560>:
>
> >          byte[] buf = new byte[65536];
> > -        try
> > +
> > +        java.io.File currFile = (java.io.File) packageMap.get(packages[i]);
> > +        if ("JAR".equals(packages[i].type) && currFile.getName().endsWith("jar"))
>
> You don't actually save the file with an extension :) I originally had
> .jar but then noticed that you are stripping most characters including dot,
> the file never has an extension. I could test the url??
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/sagetv/pull/169/files/e9d54339e3e8bd2e1c2c4e907a0b9fd7063f6151#r77739560>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANEIDOqQhEbgdjS1Yunex3eUU1j0KopJks5qnfxugaJpZM4J2Xlr>
> .
>



-- 
Jeffrey Kardatzke
jkardatzke@google.com
Google, Inc.